fanout/fanout-service.js

```diff
@@ -21,6 +16,12 @@ require('seneca')({
   })
 
   .ready(function(){
+    this.add('info:entry', function(msg,done){
```

Comment:
don't think you need to wait for ready here, pretty sure you can just .add directly